### PR TITLE
Bump rust-url to 1.0.

### PR DESCRIPTION
**Pull request by frewsxcv**

### CODE REVIEW

File: tests/test.rs [hunk 1]
```diff
@@ -89,7 +89,7 @@ fn test_unix_connection() {
     let unix_socket_directory = unix_socket_directories.split(',').next().unwrap();
 
     let path = url::percent_encoding::utf8_percent_encode(
-        unix_socket_directory, url::percent_encoding::USERNAME_ENCODE_SET);
```
Comment (frewsxcv) [inline tests/test.rs]: `USERNAME_ENCODE_SET`, `PASSWORD_ENCODE_SET`, `USERINFO_ENCODE_SET` all got merged into `USERINFO_ENCODE_SET` (they were all the same presumably)

**Comment by frewsxcv** [comment]
Looks like we may need to bump the minimum version we test on. Is that alright @sfackler?

**Comment by sfackler** [comment]
Yep, I think just to 1.7 - it's in .travis.yml.

**Comment by frewsxcv** [comment]
Force pushed.

**Comment by sfackler** [comment]
Thanks!

**Pull request merged**